issue found on: mock.go:100:1: paramTypeCombine

an issue has been detected in source code: ./mock.go:100:1: paramTypeCombine: func(msg string, val string) could be replaced with func(msg, val string)

--- a/mock.go
+++ b/mock.go
@@ -97,7 +97,7 @@ func startService(config conf.ConfigStruct) int {
 	return ExitStatusOK
 }
 
-func printInfo(msg string, val string) {
+func printInfo(msg, val string) {
 	fmt.Printf("%s\t%s\n", msg, val)
 }
 

